### PR TITLE
Update session-replay.md

### PR DESCRIPTION
**Pull request by djmixpanel**
Ton of tiny changes over past few weeks

### CODE REVIEW

File: pages/docs/features/session-replay.md [hunk 1]
```diff
@@ -10,35 +10,35 @@ When digging into customer journeys in Mixpanel’s analytics, you can understan
 
 Currently, Session Replay is currently in invite-only beta access for:
 
-- customers on our [Enterprise plan](https://www.notion.so/Session-Replay-Docs-v1-c32c9817ec1b49798d1ca7d1ccc5880c?pvs=21)
+- customers on our [Enterprise plan]([https://www.notion.so/Session-Replay-Docs-v1-c32c9817ec1b49798d1ca7d1ccc5880c?pvs=21](https://mixpanel.com/pricing/))
```
Comment (jakewski) [inline pages/docs/features/session-replay.md]: what's this link?

File: pages/docs/features/session-replay.md [hunk 2]
```diff
@@ -213,92 +213,72 @@ mixpanel.init('MIXPANEL-PROJECT-TOKEN', {
...
+Session Replay captures the Document Object Model (DOM) structure and changes to it. Mixpanel then reconstructs the web page, applying recorded events at the time an end user completed them. Within Mixpanel’s platform, you can view a reconstruction of your end user’s screen as they navigate your website. However, Session Replay is not a video recording of your end user’s screen and end user actions are not literally video-recorded.
 
 ### How does masking and blocking work? What are the high-level technical details?
 
-Masked data is suppressed client-side, meaning it is not collected in its original form by Mixpanel’s SDK, and the data is not stored on Mixpanel servers. Masked data appears in Mixpanel as [****].
+Masking and blocking are slightly different.
 
-Blocked data is similarly suppressed client-side. Blocked data will be rendered with a placeholder element (e.g., an empty box of similar size).
+Masked data is suppressed client-side, meaning it is not collected in its original form by Mixpanel’s SDK, and the data is not stored on Mixpanel servers. Masked data appears in Mixpanel as [****].
```
Comment (jakewski) [inline pages/docs/features/session-replay.md]:
```suggestion
Masked data is suppressed client-side, meaning it is not collected in its original form by Mixpanel’s SDK, and the data is not stored on Mixpanel servers. Masked elements have their text replaced with asterisks of the same length [****]. 
```

File: pages/docs/features/session-replay.md [hunk 3]
```diff
@@ -213,92 +213,72 @@ mixpanel.init('MIXPANEL-PROJECT-TOKEN', {
...
+### Configuring Privacy Controls
 
-`<div class="text.rr-mask">This text is masked!</div>`
+By default, Mixpanel masks and/or blocks the most common elements that contain content like input text, non-input text, images, and videos. However, Mixpanel also offers its customers a range of privacy controls to choose to unmask / unblock elements as needed, which are detailed further on this page. 
 
-`<img src="https://image.com" class="rr-block"/>`
+| Element Type | Default State | Customizable |
+| --- | --- | --- |
+| Inputs | Mixpanel attempts to mask all user input text.  When a user enters text into an input field, Session Replay captures [****] in place of text. | No. You cannot disable this privacy feature. |
+| Text | By default, Mixpanel attempts to mask all non-input text on your webpage. This masked content on your webpage is replaced with [****]. | Yes. Mixpanel empowers its customers to decide to record all non-input text as-is. First, change record_mask_text_selector’s default value from “*” to “” to make all text elements no longer masked. Then, you can individually mask an individual text element detailed in the next section. |
```
Comment (jakewski) [inline pages/docs/features/session-replay.md]: possible nitpick
```suggestion
| Text | By default, Mixpanel attempts to mask all non-input text on your webpage. This masked content on your webpage is replaced with [****]. | Yes. Mixpanel empowers its customers to decide to record all non-input text as-is. First, change record_mask_text_selector’s default value from “*” to “” to make all text elements no longer masked. Then, you can individually mask each text element detailed in the next section. |
```

**Review by jakewski — COMMENTED** [review]
generally looks good, just that broken link and a couple nitpicks (feel free to ignore the nitpicks)

**Review by jakewski — APPROVED** [review]

**Review by Tofufu — APPROVED** [review]

**Pull request merged**